offline_data_crop.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.image as img
+import cv2
+import os
+from core.utils import draw_gt_bbox
+
+###################################################################################
+## This script preorganizes the Taggings of Yolov3 baseline 2 format [f_data_in_path]
+## into a desired [patch_w, patch_h] cropped Tagging file [f_data_out_path] and images [write_image_path]
+## main dir of cropped outputs
+
+###################################################################################
+
```

Comment:
same comments as for rare_category_crop.py:
1. script template and configurable params
1. /home/tamar/